docs/contribute/testing/_category_.json

```diff
@@ -0,0 +1,8 @@
+{
+  "label": "Testing",
+  "position": 4,
+  "link": {
+    "type": "generated-index",
+    "description": "We will learn how to test WasmEdge applications"
```

Comment:
For the contributor guide, I think we should learn how to test WasmEdge itself, not the WasmEdge applications.

Reply:
@alabulei1 As discussed, how abt if we move this so that users can know how to test wasmedge apps?

If so, could you please suggest a subheading where I can move this to?